app/models/releases/train/run.rb

```diff
@@ -277,15 +278,14 @@ def close_pre_release_prs
     end
   end
 
-  # since we do not currently support staged-rollouts on non-production channels
-  # this check internally assumes production
-  def staged_rollout_in_progress?
-    latest_store_release&.rollout_started?
+  def version_bump_required?
```

Comment:
Think we should add a comment here explaining the differences in rules across stores.